nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -500,17 +500,17 @@ def _list_outputs(self):
             outputs["yhat"] = os.path.join(pvcdir, "yhat.nii.gz")
         if self.inputs.save_yhat_full_fov:
             outputs["yhat_full_fov"] = os.path.join(pvcdir, "yhat.fullfov.nii.gz")
-        if isdefined(self.inputs.save_yhat_with_noise) and self.inputs.save_yhat_with_noise:
+        if self.inputs.save_yhat_with_noise:
```

Comment:
@mnoergaard Sorry, I meant to include these as a separate commit from the black restyle. Hope you're fine with this change. To be clear, this works because `bool(Undefined) == False`.

Reply:
Arh, I see - thanks @effigies!